pkg/apis/networking/types.go

```diff
@@ -138,10 +138,19 @@ type NetworkPolicyPort struct {
 	// +optional
 	Protocol *api.Protocol
 
-	// The port on the given protocol. This can either be a numerical or named port on
-	// a pod. If this field is not provided, this matches all port names and numbers.
+	// The port on the given protocol. This can either be a numerical or named
+	// port on a pod. If this field is not provided, this matches all port names and
+	// numbers.
+	// If present, only traffic on the specified protocol AND port will be matched.
 	// +optional
 	Port *intstr.IntOrString
+
+	// If set, indicates that a range of ports, from port to endPort inclusive
```

Comment:
super nit: "indicates that the range of ports from port to endPort, inclusive, should be"

Reply:
I really spent something like 3 minutes to figure out that the suggestion is to change "that a range" to "that the range" and the commas :)

Replaced the whole string here as suggested :)